rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -604,10 +604,9 @@ rocalResizeMirrorNormalize(
         // RPP doesn't support returning float buffers so passing 0 and 1 as mean and std and doing normalization in rocAL
         // TODO: To be removed with rocAL Tensor support
         // rmn_node->init(0, 1, mirror);
```

Comment:
Is the commented statements necessary? Remove them if not

Reply:
Removed in the latest commit